timelock-agent/src/test/java/com/palantir/timelock/paxos/TimeLockAgentTest.java

```diff
@@ -172,6 +174,52 @@ public void newServiceSetNoDataDirectoryExistsDoesNotThrowWhenIgnoreFlagSet() {
                 .doesNotThrowAnyException();
     }
 
+    @Test
+    public void throwIfStartedWithLessThanThreeServers_DangerousTopologyNotEnabled() {
+        DefaultClusterConfiguration dangerousTopologyConfig = ImmutableDefaultClusterConfiguration.builder()
+                .localServer(SERVER_A)
+                .cluster(PartialServiceConfiguration.of(ImmutableList.of(SERVER_A, SERVER_B), Optional.empty()))
+                .build();
+        assertThatThrownBy(() -> TimeLockAgent.verifyTopologyOffersHighAvailability(
+                        TimeLockInstallConfiguration.builder()
+                                .paxos(createPaxosInstall(false, true, false))
+                                .build(),
+                        dangerousTopologyConfig))
+                .isInstanceOf(SafeIllegalArgumentException.class);
+    }
+
+    @Test
+    public void doNotIfStartedWithLessThanThreeServers_DangerousTopologyEnabledInInstallConfig() {
```

Comment:
nit: let's just name this and the next one `doesNotThrow...` (in isolation it's not super obvious what's being tested I think)